rectype,issueid,project_owner,project_name,actor,time,text,action,title,clean_text
rectype,issueid,project_owner,project_name,actor,time,text,action,title,text
issue_title,32,nilearn,nilearn,GaelVaroquaux,2012-11-19 09:18:56,"An example (with a screwed up proxy setting):

<pre>
plotting plot_visualization.py
Downloading data from http://www.pymvpa.org/files/pymvpa_exampledata.tar.bz2 ...
HTTP Error: HTTP Error 407: Proxy Authentication Required http://www.pymvpa.org/files/pymvpa_exampledata.tar.bz2
An error occured, abort fetching
extracting data from None...
archive corrupted, trying to download it again
Downloading data from http://www.pymvpa.org/files/pymvpa_exampledata.tar.bz2 ...
HTTP Error: HTTP Error 407: Proxy Authentication Required http://www.pymvpa.org/files/pymvpa_exampledata.tar.bz2
extracting data from None...
________________________________________________________________________________
plot_visualization.py is not compiling:
Traceback (most recent call last):
  File ""/home/varoquau/dev/nisl/tutorial/doc/sphinxext/gen_rst.py"", line 308, in generate_file_rst
    execfile(os.path.basename(src_file), my_globals)
  File ""plot_visualization.py"", line 10, in <module>
    haxby_files = datasets.fetch_haxby_simple()
  File ""/home/varoquau/dev/nisl/tutorial/nisl/datasets.py"", line 480, in fetch_haxby_simple
    resume=resume)
  File ""/home/varoquau/dev/nisl/tutorial/nisl/datasets.py"", line 376, in _fetch_dataset
    _uncompress_file(full_name)
  File ""/home/varoquau/dev/nisl/tutorial/nisl/datasets.py"", line 201, in _uncompress_file
    data_dir = os.path.dirname(file)
  File ""/usr/lib/python2.7/posixpath.py"", line 120, in dirname
    i = p.rfind('/') + 1
AttributeError: 'NoneType' object has no attribute 'rfind'
________________________________________________________________________________
</pre>

The traceback is not the right one
",start issue,Dataset downloaders don't fail graciusly,An exampl screw proxi set preplot plotvisualizationpi download data http error http error 407 proxi authent requir An error occur abort fetch extract data none archiv corrupt tri download download data http error http error 407 proxi authent requir extract data none plotvisualizationpi compil traceback recent call last file homevaroquaudevnisltutorialdocsphinxextgenrstpi line 308 generatefilerst execfileospathbasenamesrcfil myglob file plotvisualizationpi line 10 modul haxbyfil datasetsfetchhaxbysimpl file homevaroquaudevnisltutorialnisldatasetspi line 480 fetchhaxbysimpl resumeresum file homevaroquaudevnisltutorialnisldatasetspi line 376 fetchdataset uncompressfilefullnam file homevaroquaudevnisltutorialnisldatasetspi line 201 uncompressfil datadir ospathdirnamefil file usrlibpython27posixpathpi line 120 dirnam prfind 1 attributeerror nonetyp object attribut rfind pre the traceback right one
issue_closed,32,nilearn,nilearn,AlexandreAbraham,2013-05-02 20:02:44,,closed issue,Dataset downloaders don't fail graciusly,
issue_comment,32,nilearn,nilearn,GaelVaroquaux,2013-05-02 17:48:43,"On Thu, May 02, 2013 at 07:39:27AM -0700, Alexandre Abraham wrote:

> Can we close this issue now and maybe open it again later if someone complains
> about error messages ?

Yes. Thanks!
",,,On thu may 02 2013 073927am 0700 alexandr abraham wrote ye thank
